user/src/main/java/org/tessell/widgets/HasRows.java

```diff
@@ -23,6 +24,12 @@
   /** Removes row {@code view}. */
   boolean removeRow(final IsWidget view);
 
+  /** Assumes {@code row} is a table row and returns its index */
+  int indexOfRow(final Widget row);
```

Comment:
Given the interface is primarily used for testing (e.g. the DOM-decoupled IsWidget interface), I don't think we need this indexOfRow(Widget) overload in HasRows.